ext/DynaLoader/DynaLoader_pm.PL

```diff
@@ -292,12 +298,14 @@ sub croak   { require Carp; Carp::croak(@_)   }
 
 sub bootstrap_inherit {
     my $module = $_[0];
-    local *isa = *{"$module\::ISA"};
-    local @isa = (@isa, 'DynaLoader');
+
+    no strict 'refs';
+    push @{"$module\::ISA"}, 'DynaLoader';
     # Cannot goto due to delocalization.  Will report errors on a wrong line?
```

Comment:
For the record I believe this comment's second concern is unwarranted, Carp does not consider calls from within the same package when reporting errors.

Reply:
This seems out of scope for this work. I'm happy to take a follow on commit or help with a different PR.